{{ cookiecutter.format }}/briefcase.toml

```
@@ -6,4 +6,6 @@ target_version = "0.3.21"
 app_path = "app"
 app_requirements_path = "requirements.txt"
 
-icon.32 = "www/static/logo-32.png"
+[icon]
+"32" = "www/static/logo-32.png"
+social = "www/static/social-image.png"
```

While the `[]` format is equivalent to the dot-notation form, the dot-notation form is consistent with how we define icons on other templates.
```suggestion
icon.32 = "www/static/logo-32.png"
icon.social = "www/static/social.png"
```